keyboards/handwired/tractyl_manuform_ct/5x6_right/5x6_right.c

```diff
@@ -0,0 +1,379 @@
+/* Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+    This file contains the logical key matrix's specific C code.
+*/
+
+#include "5x6_right.h"
+#include "christrotter.h"
```

Comment:
User files shouldn't be included in keyboard files.  If you compile the default keymap, this will likely fail.
```suggestion
```